test/models/user_test.rb

```diff
@@ -2,10 +2,18 @@
 
 class UserTest < ActiveSupport::TestCase
 
-  # test "the truth" do
-  #   user = users(:daniel)
-  #   puts User.first
-  #   assert_equal true, user.valid?
-  # end
+  test "valid user saved" do
```

Comment:
again, this test doesn't add much value. and it will break if a new validation is added.